### ISSUE TITLE
Reach False Positive

### ISSUE THREAD

**Issue by thekgg** [cartridge3/nocheatminus#1]
If 2 people get hit at once it could mess up NCM detecting reach hackers.

**Comment by cartridge3** [comment]
Hey there!

Thanks for reporting this issue. I don't have tested it with more than 2 players. But yes, you're right.
I will make some thoughts, how I can fix it. I won't close this issue until the problem isn't fixed. When anyone has an idea, please comment to this issue.

**Comment by thekgg** [comment]
Depending on the amount of players you could put someone random on a team
"reachTest", then remove them over and over. Not sure if this is a good
idea or not...

**Comment by thekgg** [comment]
My only other idea is not to do reach and just normal killaura. After a
second or so reset the playerHIt score. Then do a number of tests yourself
to see what the highest number you can get. Add one to that number. Have a
command that makes it so they cant hit temporarily
if they go over that number.

**Comment by cartridge3** [comment]
Yes, the check you mean is Frequency which has been added in Build 2.
I'am currently working on an Improbable-check which will flag players which are hitting more than one entity's in one tick.
